unraid_mcp/tools/docker.py

```
@@ -1,388 +1,326 @@
+
+def register_docker_tool(mcp: FastMCP) -> None:
+    """Register the unraid_docker tool with the FastMCP instance."""
 
     @mcp.tool()
-    async def manage_docker_container(container_id: str, action: str) -> dict[str, Any]:
-        """Starts or stops a specific Docker container. Action must be 'start' or 'stop'.
+    async def unraid_docker(
+        action: DOCKER_ACTIONS,
+        container_id: str | None = None,
+        network_id: str | None = None,
+        confirm: bool = False,
```

Ruff (0.15.0)

[error] Boolean-typed positional argument in function definition

(FBT001)